.github/workflows/build_publish_lambda_layer_aarch64.yaml

```diff
@@ -0,0 +1,96 @@
+      instance-id: ${{ steps.launch.outputs.instance-id }} # ec2 instance id
+    steps:
+      - uses: getsentry/action-github-app-token@v2
+        id: github-token
+        with:
+          app_id: ${{ vars.APPLICATION_ID }}
+          private_key: ${{ secrets.APPLICATION_PRIVATE_KEY }}
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.API_CI_GHA_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.API_CI_GHA_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - id: launch
+        uses: solarwinds/ec2-runner-action@main
+        with:
+          action: launch
+          github-token: ${{ steps.github-token.outputs.token }}
+          runner-user: github
+          runner-directory: /gh
+          instance-type: t4g.medium
+          ami-name: gha-arm64-ubuntu22-.*
+          ami-owner: "858939916050"
+          subnet-id: subnet-0fd499f8a50e41807
+          security-group-ids: sg-0fd8d8cd6effda4a5
+
+  build_layer_aarch64:
+    runs-on: ${{ needs.launch_arm64.outputs.label }}
+    needs:
+      - launch_arm64
+    outputs:
+      SW_APM_VERSION: ${{ steps.save-apm-python-version.outputs.SW_APM_VERSION }}
+    steps:
+    - uses: actions/checkout@v3
+    - uses: ./.github/actions/package_lambda_solarwinds_apm_aarch64
+    - name: Save APM Python Version for naming
+      id: save-apm-python-version
+      run: |
+        echo "SW_APM_VERSION=$(grep __version__ ./solarwinds_apm/version.py | cut -d= -f 2 | tr -d ' "')" >> $GITHUB_OUTPUT
+    - uses: actions/upload-artifact@v3
+      name: Save assembled layer to build
+      with:
+        name: solarwinds_apm_lambda.zip
+        path: dist/solarwinds_apm_lambda.zip
+
+  terminate_arm64:
+    name: Terminate ec2 instances
+    if: ${{ always() }}
+    needs:
+      - launch_arm64
+      - build_layer_aarch64
+    runs-on: ubuntu-latest
+    steps:
+      - uses: getsentry/action-github-app-token@v2
+        id: github-token
+        with:
+          app_id: ${{ vars.APPLICATION_ID }}
+          private_key: ${{ secrets.APPLICATION_PRIVATE_KEY }}
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-access-key-id: ${{ secrets.API_CI_GHA_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.API_CI_GHA_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - uses: solarwinds/ec2-runner-action@main
+        with:
+          action: terminate
+          github-token: ${{ steps.github-token.outputs.token }}
+          label: ${{ needs.launch_arm64.outputs.label }}
+          instance-id: ${{ needs.launch_arm64.outputs.instance-id }}
+
+  publish-layer:
+    uses: ./.github/workflows/publish_lambda_layer.yaml
+    needs:
+      - build_layer_aarch64
+    with:
+      artifact-name: solarwinds_apm_lambda.zip
+      component-version: ${{ needs.build_layer_aarch64.outputs.SW_APM_VERSION }}
+      architecture: arm64
+      runtimes: "python3.8 python3.9 python3.10 python3.11"
```

Comment:
Two notes:

(1)
On this branch, only py3.7 and py3.8 actually have extensions built. Support for 3.9+ will be added in https://github.com/solarwinds/apm-python/pull/215 i.e. I'm prematurely adding them here.

(2)
py3.7 is not compatible with arm64, but it is with x86_64: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html